compose/base.env

```diff
@@ -29,4 +29,6 @@ DJANGO_EMAIL_PORT=1025
 # Teleport jwplatform credentials and configuration from host environment.
 JWPLATFORM_API_KEY=${JWPLATFORM_API_KEY}
 JWPLATFORM_API_SECRET=${JWPLATFORM_API_SECRET}
-JWPLATFORM_EMBED_PLAYER_KEY=${JWPLATFORM_EMBED_PLAYER_KEY}
```

Comment:
Why was this line removed? I think smswebapp.settings.developer attempts to read this from the environment? (As I read the file, it should warn about this being unset, yes?)